packages/react-components/components/Button.v2.tsx

```diff
@@ -129,3 +126,10 @@ function getStyle(size: BtnSizes | undefined, backgroundColor: Colors) {
       return { ...styles.button, ...styles.medium, backgroundColor }
   }
 }
+
+function getStyleForWrapper(
+  size: BtnSizes | undefined,
+  style: StyleProp<ViewStyle>
+): StyleProp<ViewStyle> {
+  return [{ flexDirection: size === BtnSizes.FULL ? 'column' : 'row' }, style]
+}
```

Comment:
This doesn't need to be addressed in the PR but I think the current `BtnSize` API is limiting.
In another PR I needed a button having both `FULL` (full width) and `SMALL` (40pt height) behaviors.
I don't have a proposition right now but wanted to raise this.

Reply:
Yeah, I was thinking about this as well to split this into two verticals: font size and horizontal filling. And started to think that this is needed, when I saw your workaround on this, when you were needed full width with small font size.